### ISSUE TITLE
Failed to set up: could not authenticate

### ISSUE THREAD

**Issue by Pegi68** [benleb/surepetcare#33]
Hi Ben

The Integration won't start because of an error "Failed to set up: could not authenticate".
I go to the website and could login without problem.

System: core-2022.6.3 / Python 3.9.12

Home Assistant log:
2022-08-14 18:12:06 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration sureha which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-08-14 18:12:13 DEBUG (MainThread) [custom_components.sureha] Finished fetching sureha_sensors data in 3.591 seconds (success: False)
2022-08-14 18:12:13 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Sure Petcare' for sureha integration could not authenticate_

Thanks, Horst

**Comment by bscheutjens** [comment]
I have the same issue. At first I thought it was an outage on the server side like a few weeks ago, but the app is working perfectly fine

**Comment by bscheutjens** [comment]
@Pegi68, I tried to remove the integration and add it again and that seems to work, maybe it will work for you as well

**Comment by Pegi68** [comment]
@bscheutjens yes it works like the last time i have to reinstall and reconfigure all 
Anyone know the problem because it works only for a few months?
And now I get an other problem with the battery sensor they not work :-(

I could see an error in the log...
`2022-08-17 19:08:58.972 DEBUG (MainThread) [surepy.entities.devices] error while calculating battery level: unsupported operand type(s) for -: 'NoneType' and 'NoneType'`

I'm happy that it work now but the battery state is also important for me.

**Comment by Pegi68** [comment]
Hello
Same Problem this week and only a reinstall helped to get it to work again.
By the way, thanks for the great integration 🥇 
Pegi
